src/3_PB/ManualeUtente/content.typ

```diff
@@ -0,0 +1,433 @@
+
+- Massimo livello di polveri sottili nell'aria tra quelli riportati da tutti i sensori cumulativamente, negli ultimi 5 minuti.
+
+#figure(
+image("assets/screenshots/media-e-massimo-inquinamento.png",width:50%),
+caption: [Grafico in formato Stat del livello massimo e medio di polveri sottili nell'aria.]
+)
+
+#pagebreak()
+== Dashboard "Urbanistica"
+
+La #glossary[dashboard] "Urbanistica" è dedita alla visualizzazione di dati relativi alla città, che possono includere ma non sono limitati a disponibilità dei parcheggi, disponibilità delle colonne di ricarica per automobili elettriche, stato di congestione delle strade, livello di batteria delle biciclette elettriche e riempimento delle zone ecologiche. Quelli riportati sono aspetti monitorati all'interno della #glossary[dashboard] tramite #glossary[pannelli] contenenti grafici di varia natura, in particolare si utilizza: il formato "Geomap" e il formato "Table".
+
+=== #glossary[Pannelli] con grafici in formato "Geomap"
+
+I #glossary[pannelli] contenenti grafici in formato "Geomap" riportano dati relativi a:
+
+- Disponibilità dei parcheggi, tramite un indicatore numerico che riporta il numero di posti liberi;
+
+#figure(
+image("assets/screenshots/mappa-parcheggi.png",width:100%),
+caption: [Grafico in formato Geomap della disponibilità dei parcheggi.]
+)
+
+- Disponibilità delle colonne di ricarica, tramite icone colorate;
+
+- Stato di congestione delle strade, tramite gli stati "LOW", "MEDIUM", "HIGH" e "BLOCKED";
+
+#figure(
+image("assets/screenshots/mappa-traffico.png",width:100%),
+caption: [Grafico in formato Geomap dello stato di congestione delle strade.]
+)
+
+- Livello di batteria delle biciclette elettriche, tramite un indicatore numerico che ne riporta la percentuale;
+
+#figure(
+image("assets/screenshots/mappa-biciclette.png",width:100%),
+caption: [Grafico in formato Geomap del livello di batteria delle biciclette elettriche.]
+)
+
+- Riempimento delle zone ecologiche, tramite un indicatore numerico che riporta la percentuale di riempimento.
+
+=== #glossary[Pannelli] con grafici in formato "Table"
+
+Il #glossary[pannello] contenente un grafico in formato "Table" riporta dati relativi a:
+
+- Erogazione delle colonne di ricarica riportata da ciascun #glossary[sensore], espressa in chiloWatt per ora (kWh).
+
+Tale #glossary[pannello] contiene una tabella avente campi che indicano il nome del #glossary[sensore] e l'erogazione più recente.
+
+#pagebreak()
+== Dashboard "Dati grezzi"
+
+La #glossary[dashboard] "Dati grezzi" è dedita alla raccolta e alla visualizzazione dei dati inviati da tutti i sensori e comprende sia i dati storici che quelli inviati in tempo reale; la #glossary[dashboard] consente anche di filtrare tutti i dati in base al nome e alla tipologia dei sensori di interesse, per poter risalire con facilità ai dati inviati da questi ultimi. Per questo scopo, viene utilizzato un unico #glossary[pannello] di tipo "Table".
```

Comment:
"dati storici che quelli inviati in tempo reale;" Cosa si intende dire con questa frase?

Reply:
sai che non lo so bene neanch'io, suppongo volessi dire che puoi visualizzare tutti i dati inviati dai sensori dall'alba dei tempi fino agli ultimi disponibili in tempo reale ma è strana come frase; come potrei riformularlo?

Reply:
Qualcosa tipo: "e permette di visualizzare tutti i dati inviati da i sensori e registrati nel sistema"